packages/client/src/index.ts

```diff
@@ -14,37 +14,27 @@
  * limitations under the License.
  */
 
-export type { Client } from "./Client.js";
-export { createClient } from "./createClient.js";
-export type { InterfaceObjectSet, ObjectSet } from "./objectSet/ObjectSet.js";
-export type { ObjectSetListener } from "./objectSet/ObjectSetListener.js";
-export type { PageResult } from "./PageResult.js";
-
-// FIXME: Should this be Objects or Object?
-export * as Objects from "./object/index.js";
-export type { OsdkObject } from "./OsdkObject.js";
-export type { Osdk } from "./OsdkObjectFrom.js";
-
-export type { WhereClause } from "./query/WhereClause.js";
-
 export { createClientContext } from "@osdk/shared.net";
 export { isOk } from "@osdk/shared.net";
 export type { ResultOrError } from "@osdk/shared.net";
 export type { ClientContext } from "@osdk/shared.net";
-
```

Comment:
Nit: were these line removals intentional?

Reply:
They were so that all the exports would sort